contractcourt/channel_arbitrator.go

```diff
@@ -824,7 +824,7 @@ func (c *ChannelArbitrator) stateStep(
 	case StateContractClosed:
 		// First, we'll fetch our chain actions, and both sets of
 		// resolutions so we can process them.
-		chainActions, err := c.log.FetchChainActions()
+		_, err := c.log.FetchChainActions()
```

Comment:
Commit message says we reconstruct chain actions, but here they are still fetched from the log.

Reply:
They're still fetched, but we don't use them.

Reply:
To me that makes the commit not completely logical. Why would you fetch them if not used?

Reply:
It makes the commit diffs smaller, of course I could just do everything in a single commit. If you follow the commits step by step, it shows that we start to reconstruct, then no longer write them all together.

Reply:
My preference (it is a preference) is to have one logical commit that makes the change, keeping behaviour in tact.